src/Kunstmaan/MediaBundle/Helper/Image/ImageHandler.php

```diff
@@ -18,10 +18,10 @@ class ImageHandler extends FileHandler
     /**
      * @param int                              $priority
      * @param MimeTypeGuesserFactoryInterface  $mimeTypeGuesserFactory
-     * @param ExtensionGuesserFactoryInterface $extensionGuesserFactoryInterface
-     * @param string                           $aviaryApiKey                     The aviary key
+     * @param string                           $aviaryApiKey The aviary key
+     * @param ExtensionGuesserFactoryInterface|null $extensionGuesserFactoryInterface
      */
-    public function __construct($priority, MimeTypeGuesserFactoryInterface $mimeTypeGuesserFactory, ExtensionGuesserFactoryInterface $extensionGuesserFactoryInterface, $aviaryApiKey)
+    public function __construct($priority, MimeTypeGuesserFactoryInterface $mimeTypeGuesserFactory, $aviaryApiKey, $extensionGuesserFactoryInterface = null)
```

Comment:
Changing the constructor parameter order is a BC break, also the `$extensionGuesserFactoryInterface ` parameter should not be nullable but the `@mime.types` services should be injected instead.